ServerCore/Pages/Teams/Create.cshtml.cs

```diff
@@ -70,7 +70,7 @@ public async Task<IActionResult> OnPostAsync()
                 return NotFound();
             }
 
-            if (await (from member in _context.TeamMembers
+            if (EventRole != EventRole.admin && await (from member in _context.TeamMembers
```

Comment:
Does this mean that admins can create and become part of multiple teams?

Reply:
No - admins create blank teams in all cases. The code was still checking to see if admins are on a team, which is pointless.